src/util/source_location.cpp

```diff
@@ -71,6 +71,20 @@ void source_locationt::merge(const source_locationt &from)
   }
 }
 
+/// get a path to the file in given source location
```

Comment:
I don't think this makes very clear what this method does. How about:
```
Get a path to the file, including working directory.
\return Full path unless the file name is empty or refers to a built-in, in which case the empty string is returned.
```

Reply:
done